DDDEastAnglia/Controllers/AccountController.cs

```diff
@@ -16,18 +17,41 @@ namespace DDDEastAnglia.Controllers
     public class AccountController : Controller
     {
         private readonly IUserProfileRepository userProfileRepository;
-
-        public AccountController(IUserProfileRepository userProfileRepository)
+        private readonly IBuild<LoginMethod, LoginMethodViewModel> loginMethodViewModelBuilder;
+        private readonly ExternalLoginsProvider externalLoginsProvider;
+        private readonly IOAuthClientInfo oAuthClientInfo;
+
+        public AccountController(IUserProfileRepository userProfileRepository, 
```

Comment:
Ok, well a façade service or two will help with later refactoring, and shouldn't be too hefty to implement, so can we try to get that in at least?